app/models/mixins/retirement_mixin.rb

```diff
@@ -257,6 +257,10 @@ def system_context_requester
       $log.info("System context defaulting to admin user because owner of #{name} (#{self.class}) not set or owner no longer found in database.")
       return User.super_admin
     end
+    if evm_owner.current_group.nil?
+      $log.info("#{name} has evm_owner but the user has no group.")
```

Comment:
Hi @d-m-u Can you copy the text that we use for the invalid owner in this log message? "System context defaulting to admin user because"
This way we'll know it's defaulting to admin user.